Code/Framework/AzCore/AzCore/Math/Quaternion.h

```diff
@@ -168,6 +168,14 @@ namespace AZ
         float NormalizeWithLengthEstimate();
         //! @}
 
+        //! Get the shortest equivalent of the rotation.
+        //! In case the w component of the quaternion is negative the rotation is > 180° and taking the longer path.
+        //! The quaternion will be inverted in that case to take the shortest path of rotation.
+        //! @{
+        Quaternion GetShortestEquivalent() const;
+        void ShortestEquivalent();
```

Comment:
the name here is slightly confusing?
Maybe a slightly more verbose `void ConvertToShortestEquivalent()` ?

Reply:
I'm following the pattern we're using for other functions (e.g. GetInverseFast/InverseFast, GetNormalized/Normalize). Convert seems to be used to get another rotation representation like axis-angle.